etools-loading-behavior.html

```diff
@@ -1,9 +1,18 @@
 <link rel="import" href="etools-loading.html">
+<link rel="import" href="scripts/lodash.html">
```

Comment:
It points to bower folder, just like the component will import polymer the same way. Yes, need lodash. In the end it gets vulcanized anyways no ?